### ISSUE TITLE
fix error when .vue files have `<style lang="stylus">` and `<style lang="postcss">`

### ISSUE THREAD

**Issue by r0skar** [stylelint/stylelint#3348]
**Describe the bug? (A clear and concise description of what the bug is)**

Using the latest vue-cli 3.0.0-beta.15 and stylelint 9.2.1

**What CSS is needed to reproduce this issue?**

e.g.
```vue
<template>
  <h1>Hello</h1>
</template>
<style lang="postcss">
h1 {
  color: red;
}
</style>
```

**What stylelint configuration is needed to reproduce this issue?**

```js
module.exports = {
  rules: {}
}
```

**Which version of stylelint are you using?**

`9.2.1`

**How are you running stylelint: CLI, PostCSS plugin, Node API?**

CLI with `cross-env NODE_ENV=production stylelint --fix **/*.{vue,css,scss,sass,less}`

**Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?**

Its related to PostCSS.

**What did you expect to happen?**

No error message.

**What actually happened (e.g. what warnings or errors you are getting)?**

```shell
Error: Cannot find module 'postcss-postcss'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:594:15)
    at Function.Module._load (internal/modules/cjs/loader.js:520:25)
    at Module.require (internal/modules/cjs/loader.js:650:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at requireSyntax (/__HDD/__persistent/dev/diefamilie.at/node_modules/stylelint/node_modules/postcss-syntax/get-syntax.js:26:19)
    at getSyntax (/__HDD/__persistent/dev/diefamilie.at/node_modules/stylelint/node_modules/postcss-syntax/get-syntax.js:41:10)
    at LocalFixer.parse (/__HDD/__persistent/dev/diefamilie.at/node_modules/stylelint/node_modules/postcss-syntax/parse-style.js:54:34)
    at parseStyle (/__HDD/__persistent/dev/diefamilie.at/node_modules/stylelint/node_modules/postcss-syntax/parse-style.js:89:39)
    at styles.sort.forEach.style (/__HDD/__persistent/dev/diefamilie.at/node_modules/stylelint/node_modules/postcss-syntax/parse-style.js:104:17)
    at Array.forEach (<anonymous>)
```

**Comment by evilebottnawi** [comment]
@r0skar what is `lang="postcss"`?

**Comment by r0skar** [comment]
Its a feature of the vue-loader in order to apply the correct CSS loader to the style tag inside a vue single file component. (see [vue pre-processors](https://vue-loader.vuejs.org/guide/pre-processors.html)). Although `postcss` is actually not needed anymore (its enabled by default), it still has to be used in order to get correct syntax highlighting with [Vetur](https://github.com/vuejs/vetur/).

**Comment by r0skar** [comment]
Its probably related to #3325 and https://github.com/vuejs/vue-cli/pull/1259 (?)

**Comment by evilebottnawi** [comment]
@r0skar we use `postcss-html` for parsing this, need fixed in `postcss-html`

**Comment by gucong3000** [comment]
> looks we should return css on postcss lang

fixed in #3258

**Comment by jeddy3** [comment]
Closing as seemingly fixed in https://github.com/stylelint/stylelint/pull/3331
